src/WebJobs.Script/Workers/Http/Configuration/CustomHandlerType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.Http
+{
+    public enum CustomHandlerType
+    {
+        Http,
+        None
```

Comment:
It might be safer to have values defined like

None = 0,
Http

So that even if None is not explicitly set, it defaults to the same

Reply:
Http should be the default. None is only for backward compatibility with HttpWorker section. Do you recommend Http = 0?

Reply:
Yeah, Http=0 then.